### PR TITLE
add test for 'yarn fetch-drupal-cache' script

### PR DESCRIPTION
**Pull request by mchelen-gov**
## Description
https://github.com/department-of-veterans-affairs/va.gov-team/issues/17093

## Acceptance criteria
- [x] fails PR if it breaks script

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by f1337** [comment]
Is change this in re: our Slack convo w/ @dginther? Specifically, your idea to have a non-blocking flag to tell you if the drupal fetch failed, so that you can debug RI's w/o having to SOCKS?

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,6 +77,22 @@ jobs:
       - store_artifacts:
           path: ./test-results
 
+  script-test:
```
Comment (U-DON) [inline .circleci/config.yml:80]: Perhaps something like `test-drupal-cache` might be more descriptive?

**Review by U-DON — APPROVED** [review]

**Comment by mchelen-gov** [comment]
> Is change this in re: our Slack convo w/ @dginther? Specifically, your idea to have a non-blocking flag to tell you if the drupal fetch failed, so that you can debug RI's w/o having to SOCKS?
> 
> (asking as much for my own cross-convo clarity, as for others' context)

@f1337 this is to help prevent PRs that break RIs: https://github.com/department-of-veterans-affairs/va.gov-team/issues/16637#issuecomment-737361844

File: .circleci/config.yml (hunk 1 above)
Comment (mchelen-gov) [reply .circleci/config.yml:80]: i was thinking there are other critical scripts that might be added to this test step, but i'm not sure what those are yet. maybe its better to have more specific name now and change later if additional scripts are added?

**Pull request merged**